src/index.ts

```diff
@@ -209,6 +210,18 @@ const setupTsRunner = (
     }
   }
 
+  if (process.versions.pnp) {
+    try {
+      const resolvedPnp = require.resolve('pnpapi')
+      if (
+        !NODE_OPTIONS?.includes(resolvedPnp) &&
+        !execArgv.includes(resolvedPnp)
+      ) {
+        execArgv = [...execArgv, `-r ${resolvedPnp}`]
```

Comment:
```suggestion
        execArgv.push(`-r ${resolvedPnp}`)
```

Is there any downside to push?

Reply:
No, should be fine to mutate.